ambry-protocol/src/main/java/com/github/ambry/protocol/FileCopyProtocolGetChunkRequest.java

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright 2019 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.protocol;
+
+import com.github.ambry.clustermap.ClusterMap;
+import com.github.ambry.clustermap.PartitionId;
+import com.github.ambry.utils.Utils;
+import java.io.DataInputStream;
+import java.io.IOException;
+import java.nio.charset.Charset;
+
+
+public class FileCopyProtocolGetChunkRequest extends RequestOrResponse{
+  private PartitionId partitionId;
+  private String fileName;
+  private long startOffset;
+  private long sizeInBytes;
```

Comment:
Let's not call it `sizeInBytes`, let's call it something else, like `length`. `sizeInBytes` is traditionally used to describe the byte size of the request and response.

Reply:
done